distributed/client.py

```diff
@@ -343,7 +343,7 @@ class Client(object):
     distributed.scheduler.Scheduler: Internal scheduler
     """
     def __init__(self, address=None, start=True, loop=None, timeout=3,
-                 set_as_default=True, scheduler_file=None):
+                 set_as_default=True, scheduler_file=None, connection_kwargs=None):
```

Comment:
We probably need to add a flag to the cli to set the default communication method